src/settings.ts

```diff
@@ -561,6 +564,9 @@ export const FIREBASE_CONFIG = {
   measurementId: MEASUREMENT_ID,
   projectId: PROJECT_ID,
   storageBucket: BUCKET_URL,
+  agoraAppId: AGORA_APP_ID,
+  agoraChannel: AGORA_CHANNEL,
+  agoraToken: AGORA_TOKEN,
```

Comment:
Not sure why these were added here? But they have nothing to do with the firebase config.

```suggestion
```